src/rank_llm/rerank/listwise/listwise_inference_handler.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from rank_llm.data import Result
+from rank_llm.rerank.inference_handler import BaseInferenceHandler
+
+
+# TODO(issue #237): Need to modify functions for this class
+class ListwiseInferenceHandler(BaseInferenceHandler):
+    def __init__(self, template: Dict[str, str]):
+        super().__init__(template)
+
+    def _validate_template(self, template: Dict[str, str]):
+        pass
+
+    def _generate_prefix(
+        self, num: Optional[int] = None, query: Optional[str] = None
+    ) -> str:
+        pass
+
+    def _generate_suffix(
+        self, num: Optional[int] = None, query: Optional[int] = None
+    ) -> str:
+        pass
+
+    def _generate_body(self, result: Result) -> str:
+        pass
+
+    def generate_prompt(
```

Comment:
should be the same signature as the abstract method , in body get rank_start and the rest from the kwargs